tracing04-xdp-tcpdump/README.org

```diff
@@ -29,7 +29,7 @@ struct {
 } my_map SEC(".maps");
 #+end_example
 
-The value_size determines the size of the event data we will
+The value determines the size of the event data we will
```

Comment:
The rest of this paragraph doesn't really make sense either (the "sizeof(u32)" reference). Care to update the whole thing? :)

Reply:
Hello tohojo, thank you for the review.

Nice catch!
I added a new commit to try to fix the `value` description.

Personally, I think it would be better to update the map definition in the code to use `__type(value, struct S)` instead of relying on the fact that the size of `struct S` is `__u32` (which can be kind of confusing to explain)

But, for now, I only changed the README. Let me know what you think

Reply:
Yes, absolutely agreed, this should be `__type(value, struct S)` - care to fix that as well while you're at it? :)

Reply:
Nice to hear!
I sent a new version